app/resource-center/OpenTelemetry/OpenTelemetry.tsx

```diff
@@ -32,7 +32,7 @@ export default function OpenTelemetry() {
   const posts = allCoreContent(sortPosts(allOpentelemetries))
 
   const primaryFeaturedBlogs = posts.slice(0, 2)
-  const secondaryFeaturedBlogs = posts.slice(2, 12)
+  const secondaryFeaturedBlogs = posts.slice(2, 200)
```

Comment:
Initially we had set an arbitrary number, but this should be equal to the total number of blogs

Reply:
done